Doppler.Sap/Models/SapPaymentInvoiceModel.cs

```diff
@@ -0,0 +1,10 @@
+namespace Doppler.Sap.Models
+{
+    public class SapPaymentInvoiceModel
+    {
+        public int LineNum { get; set; }
```

Comment:
This property is not used, can you remove it?

Reply:
Now is used in the mapper.